### ISSUE TITLE
Improve error message for Orbit.from_body_ephem(Pluto)

### ISSUE THREAD

**Issue by shreyasbapat** [poliastro/poliastro#401]
Adds to #382  

🐞 **Problem**

We can't plot pluto using the ephimerids data.

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~/.local/lib/python3.6/site-packages/astropy/coordinates/solar_system.py in _get_body_barycentric_posvel(body, time, ephemeris, get_velocity)
    233                 try:
--> 234                     body_index = PLAN94_BODY_NAME_TO_PLANET_INDEX[body]
    235                 except KeyError:

KeyError: 'pluto'

During handling of the above exception, another exception occurred:

KeyError                                  Traceback (most recent call last)
<ipython-input-16-3b386b096513> in <module>()
      1 earth = Orbit.from_body_ephem(Earth)
----> 2 plu = Orbit.from_body_ephem(Pluto)

~/GitHub Clones/poliastro/src/poliastro/twobody/orbit.py in from_body_ephem(cls, body, epoch)
    163                  "{}. Use Time(..., scale='tdb') instead."
    164                  .format(epoch.tdb.value), TimeScaleWarning)
--> 165         r, v = get_body_barycentric_posvel(body.name, epoch)
    166         return cls.from_vectors(body.parent, r.xyz.to(u.km), v.xyz.to(u.km / u.day), epoch)
    167 

~/.local/lib/python3.6/site-packages/astropy/coordinates/solar_system.py in get_body_barycentric_posvel(body, time, ephemeris)
    328 
    329     """
--> 330     return _get_body_barycentric_posvel(body, time, ephemeris)
    331 
    332 

~/.local/lib/python3.6/site-packages/astropy/coordinates/solar_system.py in _get_body_barycentric_posvel(body, time, ephemeris, get_velocity)
    236                     raise KeyError("{0}'s position and velocity cannot be "
    237                                    "calculated with the '{1}' ephemeris."
--> 238                                    .format(body, ephemeris))
    239                 body_pv_helio = erfa.plan94(jd1, jd2, body_index)
    240                 body_pv_bary = body_pv_helio + sun_pv_bary``

KeyError: "pluto's position and velocity cannot be calculated with the 'builtin' ephemeris."
```

🖥 **Please paste the output of following commands**

`conda info -a`

`python -c "import poliastro.testing; poliastro.testing.test()"`

`pip freeze | grep astropy`

`pip freeze | grep poliastro`

🎯 **Goal**

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
The same would happen with the Moon, by the way. The information is in the error message: `pluto's position and velocity cannot be calculated with the 'builtin' ephemeris.`. We won't fix this in poliastro, because it's the user responsability to set a proper ephemeris, but perhaps we can improve the error message.

**Comment by Himanshu-Garg** [comment]
What error message you want to display here.

(Wrong ephemeris selected, cannot load pluto data)

Is this error alright? Where name will change according to moon,pluto,etc.

**Comment by shreyasbapat** [comment]
Maybe we can include how to change the 'bulitin' ephemeris.

**Comment by Himanshu-Garg** [comment]
So what should be the message?

**Comment by shreyasbapat** [comment]
> Maybe we can include how to change the 'bulitin' ephemeris.

Just make sure to include this! If you don't know how to do this, check out this example notebook's 5th Cell. http://docs.poliastro.space/en/stable/examples/Catch%20that%20asteroid!.html
I think rest can be discussed in the Pull Request :)

**Comment by Himanshu-Garg** [comment]
from_body_ephem(Moon) is also not working.
right?

**Comment by Juanlu001** [comment]
Finally closed in #657.
